From 6e36954a00435e28a7cf9ae3754fa7b5c890fcc4 Mon Sep 17 00:00:00 2001
From: Tomoharu Fukawa <tomoharu.fukawa.eb@renesas.com>
Date: Mon, 24 Aug 2015 12:53:56 +0900
Subject: [PATCH 76/85] staging: board: salvatorx: Export reserved CMA region
 for MMNGR kernel module

In order to get the pointer of reserved CMA region in MMNGR kernel module,
call EXPORT_SYMBOL().

Signed-off-by: Tomoharu Fukawa <tomoharu.fukawa.eb@renesas.com>
---
 drivers/staging/board/salvatorx.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/board/salvatorx.c b/drivers/staging/board/salvatorx.c
index 8569648..104ca09 100644
--- a/drivers/staging/board/salvatorx.c
+++ b/drivers/staging/board/salvatorx.c
@@ -47,6 +47,7 @@ struct cma *find_largest_nondefault_cma(void)
 }
 
 struct cma *rcar_gen3_dma_contiguous;
+EXPORT_SYMBOL(rcar_gen3_dma_contiguous);
 
 static void __init salvatorx_board_staging_init(void)
 {
-- 
1.7.9.5

